internal/command/command.go

```diff
@@ -319,7 +325,34 @@ func fyneReleaseHost(ctx Context, image containerImage) error {
 
 	err = fyneCmd.Run()
 	if err != nil {
-		return fmt.Errorf("could not package the Fyne app: %v", err)
+		return "", fmt.Errorf("could not package the Fyne app: %v", err)
 	}
-	return nil
+	return searchLocalResult(volume.JoinPathHost(workDir, "*"+ext))
+}
+
+func searchLocalResult(path string) (string, error) {
+	matches, err := filepath.Glob(path)
+	if err != nil {
+		return "", fmt.Errorf("could not find the ipa file: %v", err)
```

Comment:
I don't think this method is ipa specific

Reply:
Indeed, good catch.